### PR TITLE
Add Letsencrypt

### PR DESCRIPTION
**Pull request by QuentinPerez**

### CODE REVIEW

File: overlay/etc/update-motd.d/70-owncloud [hunk 1]
```diff
@@ -9,7 +9,7 @@ if [ -f /root/.my.cnf ]; then
     cat <<EOF
 
 MySQL is configured for user "root" with password "$(cat /root/.my.cnf | grep password | awk '{print $3}')"
-To continue the installation, open https://$(/usr/local/bin/oc-metadata --cached PUBLIC_IP_ADDRESS)/owncloud/ in your browser
+To continue the installation, open https://$(/usr/local/bin/oc-metadata --cached VOLUMES_0_SERVER_ID).pub.cloud.scaleway.com/owncloud/ in your browser
```
Comment (moul) [inline overlay/etc/update-motd.d/70-owncloud]: can you switch `oc-metadata` with `scw-metadata` ?

File: overlay/usr/local/sbin/scw-letsencrypt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+DEBIAN_FRONTEND="noninteractive"
+
+URL=$(/usr/local/bin/oc-metadata --cached VOLUMES_0_SERVER_ID).pub.cloud.scaleway.com
```
Comment (moul) [inline overlay/usr/local/sbin/scw-letsencrypt]: same here

**Comment by QuentinPerez** [comment]
Done

**Comment by moul** [comment]
LGTM

**Pull request merged**